Marlin/ultralcd_impl_HD44780.h

```diff
@@ -248,20 +258,11 @@ static void lcd_set_custom_characters(
   lcd.createChar(LCD_STR_BEDTEMP[0], bedTemp);
   lcd.createChar(LCD_STR_DEGREE[0], degree);
   lcd.createChar(LCD_STR_THERMOMETER[0], thermometer);
+  lcd.createChar(LCD_STR_UPLEVEL[0], uplevel);
```

Comment:
Instead of moving this here and removing it from below, it should instead be added…